test/bigml/test/api/source.clj

```diff
@@ -8,11 +8,13 @@
   (:use clojure.test))
 
 (defn- create-and-test [artifact & params]
-  (let [initial (api/get-final (apply source/create artifact params))
+  (let [{:keys [username api_key dev_mode] :as sticky-pars} params
```

Comment:
Well, since, as Adam pointed out, you're not using any of the variables in the destructuring, probably just as easy to say

`(let [sticky-pars (select-keys params [:username :api_key :dev_mode])]`